tests/foreman/api/test_contentviewversion.py

```diff
@@ -129,3 +133,67 @@ def test_delete_version_negative(self):
             content_view.version[0].delete()
         # Make sure that content view version is still present
         self.assertEqual(len(content_view.read().version), 1)
+
+    def test_incremental_update_puppet(self):
+        """@Test: Incrementally update a CVV with a puppet module.
+
+        @Assert: The the incremental update succeeds with no errors, and the
+        content view is given an additional version.
+
+        @Feature: ContentViewVersion
+
+        """
+        # Create a content view and add a yum repository to it. Publish the CV.
+        product = entities.Product().create()
+        yum_repo = entities.Repository(
+            content_type='yum',
+            product=product,
+        ).create()
```

Comment:
Gotcha. If it turns out that empty yum repos are illegal, I'll happily update this bit of code with [a constant](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/common/constants.py#L241-L245).